internal/collector/deployment_test.go

```diff
@@ -65,6 +65,8 @@ func TestDeploymentCollector(t *testing.T) {
 		# TYPE kube_deployment_spec_strategy_rollingupdate_max_surge gauge
 		# HELP kube_deployment_labels Kubernetes labels converted to Prometheus labels.
 		# TYPE kube_deployment_labels gauge
+		# HELP kube_deployment_revision Sequence number representing a specific revision of the deployment controller represented by the annotaion 'deployment.kubernetes.io/revision'.
```

Comment:
```suggestion
		# HELP kube_deployment_revision Sequence number representing a specific revision of the deployment controller represented by the annotation 'deployment.kubernetes.io/revision'.
```

Reply:
Fixed typo and changed from Sequence number into Number, because the revision is not really a sequence.